CONTRIBUTING.md

```diff
@@ -15,9 +15,39 @@ We welcome any type of contribution, not only code. You can help with
 
 Working on your first Pull Request? You can learn how from this *free* series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
 
+### Developing
+
+After cloning the project, run the following in your terminal
+
+```sh
+npm install
+npm run build
+npx serve .
```

Comment:
Maybe we should mention the `dev` command before this ?

like after install
`npm run dev` for streamline one you mentioned below and then this `build` thing ?

WDYT ?

cause we have the same in gitpod environment as well so for consistency we can have that.